### PR TITLE
SISRP-15118, advisor_student_overview page and initial student summary widget

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.berkeley.edu/browse/SISRP-15118

Includes:
* groundwork for Advisor's student_overview page
* `aggregated_attributes` becomes a proper feed with camelCase keys

### CODE REVIEW

File: src/assets/templates/advisor_student_overview.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<div>
```
Comment (christianvuerings) [inline src/assets/templates/advisor_student_overview.html]: @johncrossman should this be behind a feature flag?
Also, can we make sure we see a 404 page when the user doesn't have the `advisor` role?

Comment (johncrossman) [inline src/assets/templates/advisor_student_overview.html]: Thanks @christianvuerings 
* I'll drop in a new feature flag, if necessary: https://jira.berkeley.edu/browse/SISRP-16963
* I've put in the 404 logic and it matches the restriction we have in `StudentOverviewController: can_view_as_for_all_uids?`

**Comment by christianvuerings** [comment]
@johncrossman thanks for making the 404 update. I would suggest though that we add the feature flag before merging.

**Comment by johncrossman** [comment]
@christianvuerings : added feature flag, spec and controller-level check.

**Comment by christianvuerings** [comment]
@johncrossman Thanks - lgtm as soon as Travis failure (`StudentLookupFeatureFlagged not found`) is resolved.

**Comment by johncrossman** [comment]
Whoops, forgot to git-add. All in now. Thx, @christianvuerings

**Pull request merged**